.gitignore

```diff
@@ -27,3 +27,4 @@ tmp
 .vs
 cmd/olympus/bin
 cmd/proxy/bin
+.DS_Store
```

Comment:
what code editor do you use? 🆕 line

Reply:
@michalpristas  I use VS-Code but it's been very slow and not working for me :( which is why you see .idea somewhere in my other pr lol.